ui/app/helpers/constants/routes.js

```diff
@@ -18,6 +18,7 @@ const CONFIRM_ADD_SUGGESTED_TOKEN_ROUTE = '/confirm-add-suggested-token'
 const NEW_ACCOUNT_ROUTE = '/new-account'
 const IMPORT_ACCOUNT_ROUTE = '/new-account/import'
 const CONNECT_HARDWARE_ROUTE = '/new-account/connect'
+const EXTERNAL_ACCOUNT_ROUTE = '/new-account/external'
```

Comment:
anything called something like "external sign(er)" should be renamed something like "BidirectionalQrSign(er)"